lesscss/components/combatants.less

```diff
@@ -142,18 +147,56 @@
 }
 
 .combatant__tags {
+  display: flex;
   flex: 1;
   flex-flow: row wrap;
   justify-content: flex-end;
+  list-style: none;
+  margin: 0;
+  padding: 0;
 }
 
 .combatant .tag {
   padding-right: @extra-small-spacer;
   flex-shrink: 1;
+  display: flex;
+  align-items: center;
+  margin-left: 0.5rem;
 }
 
 .combatant .tag__text {
   display: block;
   flex-shrink: 1;
   text-align: right;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.combatant .tag__button {
+  flex-shrink: 1;
+  text-align: right;
+  background: transparent;
+  border: 0;
+  height: 1rem;
+  width: 1rem;
+}
+
+.combatant__commands {
+  width: 6rem;
+}
+
+.commandBtn {
```

Comment:
I'd like to buy a vowel

A better name might be combatant__command-button

Reply:
class name changed